rc/filetype/lua.kak

```diff
@@ -59,6 +59,7 @@ add-highlighter shared/lua/code/function_call regex \b([a-zA-Z_]\w*)\h*(?=[\(\{]
 # ‾‾‾‾‾‾‾‾
 
 define-command lua-alternative-file -docstring 'Jump to the alternate file (implementation ↔ test)' %{ evaluate-commands %sh{
+    altfile=""
```

Comment:
What about `$altdir`?

Reply:
Move that inside the lua case ? the default case already exits.

Reply:
Done, also added initialization of `altdir`.

I need to quote it in `[ -d "$altdir" ]` if it can be empty, because in that case it would be true - `[` with a single non-empty argument is always true.